server/www/templates/common/filter-system/filter-searchbar.html

```diff
@@ -1,9 +1,9 @@
 <div class="searchbar-input" dropdown auto-close="outsideClick">
     <!-- Search input -->
     <div class="form-field search">
-        <input name="q" type="search" 
-        placeholder="{{ 'toolbar.searchbar.search_entity' | translate:{entity: placeholderText} }}" 
-        ng-model="q">
+        <input name="q" type="search"
+        placeholder="{{ 'toolbar.searchbar.search_entity' | translate:{entity: placeholderText} }}"
+        ng-model="filters.q">
```

Comment:
I did some reading. Adding `ng-model-options="{ updateOn : 'submit' }"` to the form.. makes angular queue model updates until you click submit. Which means angular basically handles the entire submit/reset workflow for us :)